lib/puppet/type/concat_file.rb

```diff
@@ -85,40 +92,56 @@ def exists?
   end
 
   # Inherit File parameters
-  newparam(:selinux_ignore_defaults) do
-  end
+  newparam(:selinux_ignore_defaults, :boolean => true, :parent => Puppet::Parameter::Boolean)
```

Comment:
Well what do you know, *apparently* it does need a parent. Is this because it's a parameter, and only properties don't need a parent?